examples/apfl_example/client.py

```diff
@@ -1,41 +1,38 @@
 import argparse
```

Comment:
I didn't realize this until looking at it, but we can eliminate the custom `def fit` method in FedProx by defining 
```
# Store current loss which is the vanilla loss without the proximal term added in
self.current_loss = loss_dict["checkpoint"]
```
in an `update_after_train` function correct?

Reply:
Good catch, updated in [d50ae1e](https://github.com/VectorInstitute/FL4Health/pull/62/commits/d50ae1eb23a57673481fd7322694a0d0d11ed0b9)!